src/Lykke.Service.PayInternal.Core/Domain/Transfer/ITransferRepository.cs

```diff
@@ -0,0 +1,38 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Lykke.Service.PayInternal.Core.Domain.Transfer
+{
+    /// <summary>
+    /// Repository for transfer entity
+    /// </summary>
+    public interface ITransferRepository
```

Comment:
I guess ITransferRequestRepository name better describes the purpose of it, right?